src/relayserver/RelayServer.ts

```diff
@@ -16,7 +16,7 @@ import { KeyManager } from './KeyManager'
 import ContractInteractor from '../relayclient/ContractInteractor'
 import PingResponse from '../common/PingResponse'
 import { Address, IntString } from '../relayclient/types/Aliases'
-import GsnTransactionDetails from '../relayclient/types/GsnTransactionDetails'
+import TmpRelayTransactionJsonRequest from '../relayclient/types/TmpRelayTransactionJsonRequest'
```

Comment:
a reminder that this class was  supposed to be temporary until we unify client and server request types

Reply:
Yes it should be removed, I didn't think that I should refactor it in this PR